app/views/users/show.html.erb

```diff
@@ -0,0 +1,99 @@
+<%= render 'shared/menubar' %>
+<%= render 'shared/navbar' %>
+
+<% if @user == current_user %>
+  <div class="container text-center justify-content-center">
+    <h1>Hi, <%= current_user.first_name %>!</h1>
```

Comment:
remove the coma

Reply:
Virna will be happy about this, she wanted to remove the comma way earlier haha I thought it looked good, but I am overruled now :D